extension/healthcheckextension/factory.go

```diff
@@ -27,7 +27,7 @@ const (
 	// The value of extension "type" in configuration.
 	typeStr = "health_check"
 
-	defaultEndpoint = "localhost:13133"
+	defaultEndpoint = "0.0.0.0:13133"
```

Comment:
Please comment why so by mistake somebody does not revert

Reply:
Added a comment